lib/evss/mhvcf/client.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+require 'common/client/base'
+require 'evss/auth_headers'
+require 'evss/mhvcf/configuration'
+require 'evss/mhvcf/get_in_flight_forms_request_form'
+require 'evss/mhvcf/mhv_consent_form_request_form'
+
+module EVSS
+  module MHVCF
+    class Client < Common::Client::Base
+      configuration EVSS::MHVCF::Configuration
+
+      attr_accessor :auth_headers
+
+      def user(user)
```

Comment:
yes, i did this temporarily out of laziness while writing the spec, 100% agree this should be in initialization.